packages/destination-actions/src/destinations/kameleoon/group/index.ts

```diff
@@ -0,0 +1,85 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+import omit from 'lodash/omit'
+
+import { BASE_URL } from '../properties'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Group',
+  description: 'Group user in Kameleoon',
+  defaultSubscription: 'type = "group"',
+  fields: {
+    anonymousId: {
+      type: 'string',
+      description: 'Anonymous id',
+      label: 'Anonymous ID',
+      default: { '@path': '$.anonymousId' }
+    },
+    userId: {
+      type: 'string',
+      description: 'The ID associated with the user',
+      label: 'User ID',
+      default: { '@path': '$.userId' }
+    },
+    groupId: {
+      type: 'string',
+      description: 'The group id',
+      label: 'Group ID',
+      required: true,
+      default: { '@path': '$.groupId' }
+    },
+    properties: {
+      type: 'object',
+      required: false,
+      description: 'Traits to send with the event',
+      label: 'user traits',
```

Comment:
Should this say 'User Traits' or something which aligns more with what a group is?
Please also Capital Case the label. 

```suggestion
      label: 'User Traits',
```